ISICbalance/ViewControllers/BalanceViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  BalanceViewController.swift
+//  ISICbalance
+//
+//  Created by Rostislav Babáček on 12/03/2019.
+//  Copyright © 2019 Rostislav Babáček. All rights reserved.
+//
+
+import UIKit
+import SwiftKeychainWrapper
+import SnapKit
+
+class BalanceViewController: BaseViewController {
+    private let viewModel: BalanceViewModel
+    private let rm: RequestManager
```

Comment:
RequestManager tady nemá co dělat. Jsem VC a nic mě nezajímá, z VM si vezmu na míru připravený data, který zobrazím a říkám mu, že uživatel na něco kliká, ale víc nedělám. Takže nevolám rm.reload ale viewModel.reload a je mi šumák jak si to ten VM vyřeší.

Reply:
Tohle jsem dnes přepisoval reaktivně. Z metody reloadBalance spouštím akci, která je ve VM.

Reply:
Mám to opravovat?

Reply:
Jestli to řešíš někde později, tak to tady neopravuj.